src/storage/leveldb/utxo_index.ts

```diff
@@ -186,6 +186,15 @@ export default class LevelUtxoIndex implements IKVUtxoIndex {
       // Heighlocked when network height is lower than block height + reward_spend_min_blocks
       return ((utxo.height || 0) + options.reward_lock) > networkHeight;
```

Comment:
I think there's a bug here. You are never passing `networkHeight` when you use this method. https://github.com/HathorNetwork/hathor-wallet-lib/blob/dev/src/storage/storage.ts#L292

The method `selectUtxos` from memory store does not have `networkHeight` as parameter, it gets using `this.getCurrentHeight`. Both methods should have the same signature. The interface follows what's in the memory store: https://github.com/HathorNetwork/hathor-wallet-lib/blob/dev/src/types.ts#L288

Reply:
But this is the utxo index, which is internal to the leveldb store.
We get the network height and pass to this method in the [store](https://github.com/HathorNetwork/hathor-wallet-lib/blob/feat/leveldb-filter-locked-utxos/src/storage/leveldb/store.ts#L227).